### DISCUSSION TITLE
Not working with latest NVIDIA Studio driver

### DISCUSSION THREAD

**Discussion by imovietone** [mifi/lossless-cut#1665] [Support] [answered]
I have 2 Windows PC's:
Windows 10, Intel integrated graphics
Windows 11, RTX 3060Ti, latest Studio driver. (Intel cpu. No integrated graphics)
Lossles-cut was working fine on both until a couple of months ago. Same version.
Now loading, navigation and export are so slow on the NVIDIA one its virtually unusable.
I'm only working with small h.264 mp4 files.

**Comment by mifi** [comment]
If you updated your os/drivers a couple of months ago, but never did anything to losslesscut, could it be a bug in the newest NVIDIA driver?

**Reply by imovietone** [reply]
Maybe, but:
No other apps have been adversley affected, and the driver was further updated in the last fe days. I was hoping that would fix it.

**Reply by mifi** [reply]
Unfortunately I'm clueless as to what's causing this problem

**Reply by imovietone** [reply, marked as answer]
Thanks for your responses. I've moved the LC folder from Portables to Program Files and all is well again!!
